### DISCUSSION TITLE
Regarding Repository for Amazon Linux 2023

### DISCUSSION THREAD

**Discussion by manoj-source** [amazonlinux/amazon-linux-2022#308] [Q&A]
When we publish a new version of the AL2023 repositories, all previous versions will still be available. By default, the plugin for managing repository versions will lock to the same version that was used to build the AMI. If you need to control package updates, you can discover available repository versions to update to by running “dnf check-release-update”, and select a version by running the listed command, “dnf —releasever=version update”. At that point, “dnf install” or “dnf upgrade“ will only choose packages from the selected repository version. If you do not need to control package updates, you can select the “latest” version, which will always point to the most recent version of the AL2023 repositories.

Is their any AL2023 repo where all the packages will be updated, instead multiple repos  ?

**Comment by stewartsmith** [comment]
Using the `latest` as `releasever` will mean the AL2023 core repository points to the version with the latest AL2023 release.
